src/crypto/CHIPCryptoPALPSA.cpp

```diff
@@ -1684,6 +1685,98 @@ CHIP_ERROR ExtractAKIDFromX509Cert(const ByteSpan & certificate, MutableByteSpan
     return ExtractKIDFromX509Cert(false, certificate, akid);
 }
 
+CHIP_ERROR ExtractCRLDistributionPointURIFromX509Cert(const ByteSpan & certificate, MutableCharSpan & cdpurl)
```

Comment:
Would it be possible to move all x509 common code for mbedtls in a new `.ipp` file that is includable from all mbedtls-based CryptoPALs ? The duplication is a lot and keeps growing.

Reply:
Added ticket to follow up on this: #27831